index.html

```diff
@@ -0,0 +1,299 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap" rel="stylesheet">
+</head>
+  <title>Document</title>
+</head>
+<body>
+  <header>
+    <div class="nav-container">
+      <div class="nav-logo">
+        <a style="text-decoration: none; color: blue;" href="#">My Logo</a>
+      </div>
+      <div class="desktop-tools">
+        <nav class="content">
+          <ul class="navnav-menu">
+            <li class="navnav-item">
+              <a href="#Portfolio" class="link">Portfolio</a>
+            </li>
+            <li>
+              <a href="#About" class="link">About</a>
+            </li>
+            <li>
+              <a href="#Contact" class="link">Contact</a>
+            </li>
+          </ul>
+        </nav>
+        <div class="hamburger">
+          <span class="barbar"></span>
+          <span class="barbar"></span>
+          <span class="barbar"></span>
+        </div>
```

Comment:
- [ ] For semantic reasons I highly recommend you put your `hamburger` menu in a `nav` instead of a `div`. 🙂